plugins/node/opentelemetry-instrumentation-aws-sdk/test/load-instrumentation.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  initMeterProvider,
+  registerInstrumentationTesting,
+} from '@opentelemetry/contrib-test-utils';
+import { AwsInstrumentation } from '../src';
+
+export const instrumentation = new AwsInstrumentation();
+export const metricReader = initMeterProvider(instrumentation);
+instrumentation._updateMetricInstruments();
```

Comment:
This method is called in InstrumentationAbstract's constructor. Is it necessary to call it again?
The method is also called in `initMeterProvider` (that calls `instrumentation.setMeterProvider(meterProvider);`, which I think calls `instr._updateMetricInstruments()`).